tests/integ/datasource/test_oracledb.py

```diff
@@ -0,0 +1,180 @@
+#
+# Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
+#
+
+import logging
+import math
+import pytest
+
+from snowflake.snowpark import Row
+from snowflake.snowpark._internal.data_source.drivers.oracledb_driver import (
+    output_type_handler,
+)
+from snowflake.snowpark._internal.data_source.drivers import (
+    OracledbDriver,
+)
+from snowflake.snowpark._internal.data_source.utils import (
+    DBMS_TYPE,
+)
+from tests.parameters import ORACLEDB_CONNECTION_PARAMETERS
+from tests.resources.test_data_source_dir.test_data_source_data import (
+    OracleDBType,
+    oracledb_real_data,
+    oracledb_real_data_small,
+)
+from tests.utils import RUNNING_ON_JENKINS, Utils
+
+DEPENDENCIES_PACKAGE_UNAVAILABLE = True
+try:
+    import pandas  # noqa: F401
+    import oracledb  # noqa: F401
+
+    DEPENDENCIES_PACKAGE_UNAVAILABLE = True
+except ImportError:
+    pass
+
+
+pytestmark = [
+    pytest.mark.skipif(
+        "config.getoption('local_testing_mode', default=False)",
+        reason="feature not available in local testing",
+    ),
+    pytest.mark.skipif(
+        DEPENDENCIES_PACKAGE_UNAVAILABLE,
+        reason="dependency is not available",
+    ),
+    pytest.mark.skipif(
+        RUNNING_ON_JENKINS,
```

Comment:
I think we can switch to using `not RUNNING_ON_GH` -- so it prevents running on both jenkins and reg env where we don't have EAI.

Reply:
that would make the test not be able to run in local, is there also a param to know whether it is in local?

Reply:
oh you are right, never mind then